src/components/event/experimental-api/EventTree.tsx

```diff
@@ -53,6 +53,7 @@ function EventTree({ eventId }: EventTreeProps) {
 
 	const onExpandClick = () => {
 		eventsStore.toggleExpand(eventId);
+		eventsStore.clearEventChildren(eventId);
 		eventsStore.fetchChildren(eventId);
```

Comment:
Now since we have pagination i think `eventsStore.fetchChildren` should be called here only when it's expanded for the first time